crates/bevy_pbr/src/material.rs

```diff
@@ -578,8 +622,23 @@ pub fn queue_material_meshes<M: Material>(
     }
 }
 
+/// Default render method used for opaque materials.
+#[derive(Default, Resource, Clone, Debug, ExtractResource, Reflect)]
+pub struct DefaultOpaqueRendererMethod(pub OpaqueRendererMethod);
+
+/// Render method used for opaque materials.
```

Comment:
```suggestion
/// Render method used for opaque materials.
///
/// The forward rendering main pass draws each mesh entity and shades it according to its
/// corresponding material and the lights that affect it. Some render features like Screen Space
/// Ambient Occlusion require running depth and normal prepasses, that are 'deferred'-like
/// prepasses over all mesh entities to populate depth and normal textures. This means that when
/// using render features that require running prepasses, multiple passes over all visible geometry
/// are required. This can be slow if there is a lot of geometry that cannot be batched into few
/// draws.
/// 
/// Deferred rendering runs a prepass to gather not only geometric information like depth and
/// normals, but also all the material properties like base color, emissive color, reflectance,
/// metallicness, etc, and writes them into a deferred 'g-buffer' texture. The deferred main pass
/// is then a fullscreen pass that reads data from these textures and executes shading. This allows
/// for one pass over geometry, but is at the cost of not being able to use MSAA, and has heavier
/// bandwidth usage which is unsuitable for mobile and other bandwidth-constrained devices.
```

Reply:
Changed the end to `bandwidth usage which can be unsuitable for low end mobile or other bandwidth-constrained devices.` since I think deferred is fine on modern high end (and probably also middle end) mobile devices. Also changed metallicness to metalness. I think metalness is actually what is often used when talking about pbr.